Makefile

```diff
@@ -94,3 +94,14 @@ test-failpoint:
 test-robustness: gofail-enable build
 	sudo env PATH=$$PATH go test -v ${TESTFLAGS} ./tests/dmflakey -test.root
 	sudo env PATH=$(PWD)/bin:$$PATH go test -v ${TESTFLAGS} ${ROBUSTNESS_TESTFLAGS} ./tests/robustness -test.root
+
+.PHONY: test-benchmark-compare
+# Runs benchmark tests on the current git ref and the given REF, and compares
+# the two.
+test-benchmark-compare:
+	@git fetch
+	./scripts/compare_benchmarks.sh $(REF)
+
+.PHONY: install-benchstat
+install-benchstat:
+	go install golang.org/x/perf/cmd/benchstat@latest
```

Comment:
I would pin this to a given version

Reply:
I was going to raise the same comment.  In that case, we also need to maintain/upgrade the version from time to time, it's extra overhead:).

Reply:
Honestly, this comes from the original implementation (PR #691). However, I spotted something similar in:

https://github.com/etcd-io/etcd/blob/8c1b08b69ffc27d769dd9331158f9b2ee92a8e3d/.github/workflows/govuln.yaml#L19

One point is that even Go's documentation https://pkg.go.dev/golang.org/x/perf#readme-tools refer you to do it with `latest`.

So, in the end, I don't have a strong opinion or preference regarding this. I'll follow whatever the consensus is for this :)

Reply:
Depends how robust we can make that CSV logic. Then again, I'm not sure whether benchstats will change its column layout all the time

Reply:
@tjungblu, refer to https://github.com/etcd-io/bbolt/pull/750#discussion_r1619468759

Reply:
Marking as resolved for now.